baselines/common/cmd_util.py

```diff
@@ -32,15 +32,15 @@ def _thunk():
     set_global_seeds(seed)
     return SubprocVecEnv([make_env(i + start_index) for i in range(num_env)])
 
-def make_mujoco_env(env_id, seed, reward_scale=1.0):
+def make_mujoco_env(env_id, seed, i=0, reward_scale=1.0):
     """
     Create a wrapped, monitored gym.Env for MuJoCo.
     """
     rank = MPI.COMM_WORLD.Get_rank()
-    myseed = seed  + 1000 * rank if seed is not None else None
+    myseed = seed + i + 1000 * rank if seed is not None else None
     set_global_seeds(myseed)
     env = gym.make(env_id)
-    logger_path = None if logger.get_dir() is None else os.path.join(logger.get_dir(), str(rank))
+    logger_path = None if logger.get_dir() is None else os.path.join(logger.get_dir(), str(rank)+ '.' + str(i))
```

Comment:
it may make sense to make this more like make_atari_env (lines 19-33 in this file) and have num_env passed as an argument... In fact, most of the code from make_atari_env can be reused; so you can make a common make_vec_env function that does the magic with monitor, seeds etc, and accepts either wrap_deepmind (for atari) or gym.make(...) for mujoco. Does that make sense?

Reply:
I did it:
I unified make_atari_env and make_mujoco_env in one function make_vec_env. 
The easy way to do it was to pass env_type as an argument in make_vec_env.
The default behaviour for mujoco env is still to set num_env to 1 for mujoco and to 8 for atari. 
I noticed that when using nenv = 1, for atari, it is using SubProcVecEnv instead of DummyVecEnv which adds unnecessary process communication overhead as pointed out in #519 . So i corrected it. 
I am currently replacing make_mujoco_env by make_vec_env in acktr.run_mujoco .
I am also replacing make_mujoco_env by make_vec_env in ppo1.run_mujoco and ppo1.run_humanoid even though ppo1 is deprecated.
My ide automatically removed some unecessary spaces, i did not want to commit that, sorry about that. See the second commit instead of the third one.